pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/ActivityStatusEventDao.java

```diff
@@ -36,11 +36,11 @@ default int addStatusTriggerEventsToQueue(long operatorId,
                                               ActivityInstanceDto instanceDto, String status) {
         int numEventsQueued = 0;
         List<EventConfigurationDto> eventConfigs = getEventDao()
-                .getEventConfigurationsForActivityStatus(instanceDto.getId(), status);
+                .getEventConfigurationDtosForActivityStatus(instanceDto.getId(), status);
         JdbiQueuedEvent jdbiQueuedEvent = getHandle().attach(JdbiQueuedEvent.class);
         QueuedEventDao queuedEventDao = getHandle().attach(QueuedEventDao.class);
         for (EventConfigurationDto eventConfig : eventConfigs) {
-            Integer delayBeforePosting = eventConfig.getSecondsToWaitBeforePosting();
+            Integer delayBeforePosting = eventConfig.getPostDelaySeconds();
```

Comment:
While not something to address for this PR, this feels like a situations where we'd want to consider using a value-type object instead of passing around bare integers. When classes like `java.time.Duration` exist, stating the units in the methods name is a bit of an anti-pattern.

Reply:
👍